### DISCUSSION TITLE
How to disable CSS file merging in Next.js?

### DISCUSSION THREAD

**Discussion by bossman1999** [vercel/next.js#52715] [Help]
### Summary

In Next.js 13.4.9, during development mode, all CSS styles are being merged into a single CSS file called layout.css?v=1689410170225. This makes it difficult to debug CSS styles. I have tried various configurations, but none of them have worked. Is there any way to prevent CSS merging?

**Comment by robertpiosik** [comment]
Not all css are merged into one file, a file is generated per route segment, so if you nest many layouts you will get a separate css for each and one for page component.

Could you describe your debugging workflow? Many people debug in the "elements" tab of dev tools, this way, how css are split in files is irrelevant.

**Reply by realduolaf** [reply]
Hi. Is there any way to combine css files from different pages into one file？

**Reply by robertpiosik** [reply]
Without avoiding duplication, no. With duplications, you could write a post-build script to combine css files together and attach them where you want them to load.
